projects/common/src/lib/datasets/ogr-dataset/ogr-dataset.component.html

```diff
@@ -205,4 +205,28 @@
         <mat-label>Spatial Reference</mat-label>
         <input matInput type="text" formControlName="spatialReference" />
     </mat-form-field>
+
+    <mat-form-field class="column">
```

Comment:
Could you include some sort of indication in the UI that this section is meant for setting the measurement? Alternatively, would it maybe make sense to have the "geoengine-measurement" appear directly below the "Integer Columns" etc. fields when one clicks on a column name/enters a new one there? It might be more intuitive to have it directly there, associated with the column definition and would be neat to have it only appear when actively editing the column names this way and hidden otherwise.